test/Interpreter/Inputs/ObjCClasses/ObjCClasses.m

```diff
@@ -132,3 +133,15 @@ - (NSInteger) count {
   return _objects.count;
 }
 @end
+
+@implementation TestingBool
+
+- (void) shouldBeTrueObjCBool: (BOOL)value {
+  assert(value);
```

Comment:
The IRGen tests do verify that the representation is correct, so if you don't mind I'll fix up the tests in a separate patch on master.